app/ante.go

```diff
@@ -31,6 +31,7 @@ func NewAnteHandler(
 ) sdk.AnteHandler {
 	mempoolFeeOptions := txfeestypes.NewMempoolFeeOptions(appOpts)
 	mempoolFeeDecorator := txfeeskeeper.NewMempoolFeeDecorator(*txFeesKeeper, mempoolFeeOptions)
+	deductFeeDecorator := txfeeskeeper.NewDeductFeeDecorator(*txFeesKeeper, ak, bankKeeper, nil)
```

Comment:
Just an FYI, I think I mentioned to @ValarDragon that ultimately ante-handler decorators/middleware should be defined in a root `ante` package, instead of in modules. However, we can address this in a future PR 👍